AsyncDisplayKit/Layout/ASLayoutable.mm

```diff
@@ -174,10 +218,9 @@ - (void)setMaxHeight:(ASDimension)maxHeight
 {
   ASDN::MutexLocker l(__instanceLock__);
   _size.maxHeight = maxHeight;
+  ASLayoutableStyleCallDelegate(ASLayoutableStyleMaxHeightProperty);
```

Comment:
Note for the record: Michael and I discussed whether to hold the lock during this delegate call. It's a tough call, and it's unlikely to matter, but I advocated to do so on the basis that NSCache does the same, and that informing the delegate should be "part of" the same atomic operation.

Since we are currently using a recursive mutex – which may be worth changing in a class as simple as this one – it is almost certain not to make any difference.